projects/escher/index.js

```diff
@@ -19,10 +19,30 @@ async function eBabyTVL(api) {
   api.add(token, totalStakedAmount);
 }
 
+async function eUTVL(api) {
+  const lcd = 'https://rest.union.build';
```

Comment:
please use our SDK for EVM TVLs. Maybe use staking() from our helpers. Also since this is your own token, please export this as `staking` rather than `tvl` and use `tvl: () => ({})`